readme.md

```diff
@@ -28,35 +28,19 @@ Please make sure you install the following tools before starting with the instal
 
 ## Installation
 
-> Note that you're free to adjust the `~/Sites/pastebin` location to any directory you want on your machine.
+> Note that you're free to adjust the `~/Code/pastebin` location that will be in the `Homestead.yaml` to any directory you want on your machine.
 
 1. Clone this repository: `git clone git@github.com:laravelio/pastebin.git pastebin`
-2. Add the `Homestead.yaml` file from below to the root of your project
+2. Run `./vendor/bin/homestead make`
 3. Run `vagrant up`
-4. SSH into your Vagrant box, go to `/home/vagrant/pastebin` and run the following commands:
+4. SSH into your Vagrant box, go to `/home/vagrant/Code/pastebin` and run the following commands:
```

Comment:
With the current release of Homestead they will see `/home/vagrant/Code/pastebin` mapping in the VM (unless they changed that in their Homestead.yaml). I have removed the `/Code/` part of the path in https://github.com/laravel/homestead/pull/601 but I haven't yet tagged a new release. So we should leave this for now IMHO.